README.md

```diff
@@ -23,15 +22,17 @@ The Release schedule is available also as a [JSON][] file.
 
 ### End-of-Life Releases
 
-| Release | Status              | Codename   |Initial Release | Active LTS Start | Maintenance LTS Start | End-of-life                 |
-| :--:    | :---:               | :---:      | :---:          | :---:            | :---:                 | :---:                       |
-| v0.10.x | **End-of-Life**     | -          | 2013-03-11     | -                | 2015-10-01            | 2016-10-31                 |
-| v0.12.x | **End-of-Life**     | -          | 2015-02-06     | -                | 2016-04-01            | 2016-12-31                 |
-| [4.x][] | **End-of-Life**     | [Argon][]  | 2015-09-08     | 2015-10-01       | 2017-04-01            | 2018-04-30                 |
-| [5.x][] | **End-of-Life**     |            | 2015-10-29     |                  |                       | 2016-06-30                 |
-| [6.x][] | **End-of-Life** | [Boron][]  | 2016-04-26     | 2016-10-18       | 2018-04-30            | 2019-04-30                 |
-| [7.x][] | **End-of-Life**     |            | 2016-10-25     |                  |                       | 2017-06-30                 |
-| [9.x]   | **End-of-Life**     |            | 2017-10-01     |                  |                       | 2018-06-30                 |
+|  Release |      Status     |  Codename | Initial Release | Active LTS Start | Maintenance LTS Start | End-of-life |
+|:--------:|:---------------:|:---------:|:---------------:|:----------------:|:---------------------:|:-----------:|
+|  v0.10.x | **End-of-Life** |     -     |    2013-03-11   |         -        |       2015-10-01      |  2016-10-31 |
+|  v0.12.x | **End-of-Life** |     -     |    2015-02-06   |         -        |       2016-04-01      |  2016-12-31 |
+|  [4.x][] | **End-of-Life** | [Argon][] |    2015-09-08   |    2015-10-01    |       2017-04-01      |  2018-04-30 |
+|  [5.x][] | **End-of-Life** |           |    2015-10-29   |                  |                       |  2016-06-30 |
+|  [6.x][] | **End-of-Life** | [Boron][] |    2016-04-26   |    2016-10-18    |       2018-04-30      |  2019-04-30 |
+|  [7.x][] | **End-of-Life** |           |    2016-10-25   |                  |                       |  2017-06-30 |
+|   [9.x]  | **End-of-Life** |           |    2017-10-01   |                  |                       |  2018-06-30 |
+| [11.x][] | **End-of-Life** |           |    2018-10-23   |                  |        May 2019       |  2019-06-01 |
```

Comment:
```suggestion
| [11.x][] | **End-of-Life** |           |    2018-10-23   |                  |                       |  2019-06-01 |
```

v11.x had a small maintenance window but it wasn't LTS

Reply:
Oh, I didn't saw that subtlety.

Thanks :)